lib/tasks/signup.rake

```diff
@@ -0,0 +1,23 @@
+namespace :signup do
+  desc "This task seeds lago with an organisation & a user for on premise deployment"
+  task seed_organization: :environment do
+    if ENV['CREATE_ORG'].present? && ENV['CREATE_ORG'] == 'true'
```

Comment:
We try to prefix every inner environment variable by `LAGO_` to make sure it does not interfere with other application. Do you mind using the same approach?